messages.js

```diff
@@ -34,7 +34,14 @@ exports.init = function init(sbot) {
 
     if (nonce) content.nonce = nonce
 
-    if (metadata) Object.assign(content, metadata)
```

Comment:
This was the naughty line

Reply:
Yeah, if we check for nuaghty keys instead we should be good.